examples/llava/clip-quantize-cli.cpp

```diff
@@ -0,0 +1,58 @@
+#include "arg.h"
+#include "base64.hpp"
+#include "log.h"
+#include "common.h"
+#include "sampling.h"
+#include "clip.h"
+#include "llava.h"
+#include "llama.h"
+#include "ggml.h"
+
+void print_usage(int argc, char ** argv) {
```

Comment:
You can mark this function static, otherwise GCC is failing with "no previous prototypes." And, you can also consume `argc` with a simple void macro --they are warnings in fact but `-Werror` flag treats all warnings as errors.

Reply:
I have just fixed this. Thanks! :D